### PR TITLE
10jan

### PR DESCRIPTION
**Pull request by Shubhamoncehub**

### CODE REVIEW

**Comment by saxenarajat** [comment]
Why is there an empty `index.js` in the top directory?

**Comment by saxenarajat** [comment]
How are you running the files located in `Shubham-kumar/daily-learning-material` folder? Shouldn't they go in the `nodejs` folder?

**Comment by Shubhamoncehub** [comment]
Hi Sir,

I take care of the mistakes further, it wont open again


*Regards*

*Shubham Kumar* | Software Trainee at OnceHub
Let's connect at | Shubham.kumar@oncehub.com | LinkedIn
<https://www.linkedin.com/in/shubham-kumar-020536141/>


________________________________________________________


   *Find out more about OnceHub:*
   - *Career Website <http://www.oncehub.com/careers/home#openPositions>  *
   *Website <http://www.oncehub.com/> | **Blog <http://blog.oncehub.com/>*
   * | **Facebook <http://www.facebook.com/OnceHub> **| Twitter
   <http://twitter.com/oncehub> | YouTube <https://www.youtube.com/OnceHub> *

*Watch the video: About OnceHub
   <https://www.youtube.com/watch?v=QuUtDqZ7Np8&feature=youtu.be>*

File: Shubham-kumar/daily-learning-material/filter.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+function filter(x,y)
+    {   
+
+        for (i =0; i<y.length; i++)
+        {
+          
+            delete x[y[0]]
```
Comment (saxenarajat) [inline Shubham-kumar/daily-learning-material/filter.js]: This will make us lose the information. We don't want that.

Comment (Shubhamoncehub) [reply Shubham-kumar/daily-learning-material/filter.js]: I will update it soon.

**Pull request merged**